docs/src/main/asciidoc/writing-extensions.adoc

```diff
@@ -1970,6 +1970,35 @@ information about this start, in particular:
 It also provides a global context map you can use to store information between restarts, without needing to resort to
 static fields.
 
+Here is an example of a build step that persists context across live reloads:
+
+[source,java]
+----
+@BuildStep(onlyIf = {IsDevelopment.class})
+public void keyPairDevService(LiveReloadBuildItem liveReloadBuildItem, BuildProducer<DevServiceResultBuildItem> devServices) {
+
+KeyPairContext ctx = liveReloadBuildItem.getContextObject(KeyPairContext.class);    // <1>
+    if (ctx == null && !liveReloadBuildItem.isLiveReload()) {                       // <2>
+        KeyPair keyPair = KeyUtils.generateKeyPair(KEY_SIZE);
```

Comment:
I propose you replace it here with your original code to generate a key pair as these docs are neutral and references to smallrye-jwt can be confusing